Modules/System/Azure Function/AzureFunctionsImpl.Codeunit.al

```diff
@@ -0,0 +1,83 @@
+// ------------------------------------------------------------------------------------------------
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+// ------------------------------------------------------------------------------------------------
+
+codeunit 7803 "Azure Functions Impl"
+{
+    Access = Internal;
+
+    var
+        SendRequstErr: Label 'Sending the request has failed.';
+        HttpResponseErr: Label '%1.\\Response Code: %2 %3', Comment = '%1 = Default Error Message ; %2 = Status Code; %3 = Reason Phrase';
+        AuthenticationFailedErr: Label 'Authentication failed';
+        AzureFunctionCategoryLbl: Label 'Connect to Azure Functions', Locked = true;
+        RequestSucceededMsg: Label 'Request sent to Azure function succeeded: %1', Locked = true;
+        RequestFailedErr: Label 'Request sent to Azure function failed: %1', Locked = true;
+
+    [NonDebuggable]
+    procedure SendGetRequest(AzureFunctionAuthentication: Interface "Azure Functions Authentication"; QueryDict: Dictionary of [Text, Text]): Codeunit "Azure Functions Response"
+    begin
+        exit(Send(AzureFunctionAuthentication, Enum::"Http Request Type"::GET, QueryDict, '', ''));
+    end;
+
+    [NonDebuggable]
+    procedure SendPostRequest(AzureFunctionAuthentication: Interface "Azure Functions Authentication"; Body: Text; ContentTypeHeader: text): Codeunit "Azure Functions Response"
+    var
+        QueryDict: Dictionary of [Text, Text];
+    begin
+        exit(Send(AzureFunctionAuthentication, Enum::"Http Request Type"::POST, QueryDict, Body, ContentTypeHeader));
+    end;
+
+    [NonDebuggable]
+    procedure Send(AzureFunctionAuthentication: Interface "Azure Functions Authentication"; RequestType: enum "Http Request Type"; QueryDict: Dictionary of [Text, Text]; Body: Text; ContentTypeHeader: text): Codeunit "Azure Functions Response"
+    var
+        Uri: Codeunit Uri;
+        UriBuider: Codeunit "Uri Builder";
```

Comment:
There's a small typo. Should be `UriBuilder`

Reply:
If you haven't heard of it, https://marketplace.visualstudio.com/items?itemName=rasmus.al-var-helper lets you insert the variable name automatically - hope that helps, I learned about it last friday 📑 

As we are speaking about typos: there is also a missing s in the first parameter. Should be `AzureFunctionsAuthentication`